core/src/Cuda/Kokkos_Cuda_Parallel.hpp

```diff
@@ -1655,20 +1692,22 @@ class ParallelReduce<FunctorType, Kokkos::TeamPolicy<Properties...>,
       dim3 grid(block_count, 1, 1);
       const int shmem_size_total = m_team_begin + m_shmem_begin + m_shmem_size;
 
+      if ((nwork < 1)
```

Comment:
```suggestion
      if ((nwork == 0)
```
to look the same as above.

Reply:
done